.travis.yml

```diff
@@ -18,8 +18,12 @@ cache:
     - $HOME/ibm-jdk-installer/
 
 install: true
-script:
-  - ./gradlew test systemTest
+script: ./gradlew test systemTest
+
+after_failure:
+  - pip install --user awscli
+  - if [ "${TRAVIS_REPO_SLUG}" = "inspectIT/inspectit-oce" ]; then aws s3 cp $TRAVIS_BUILD_DIR/inspectit-oce-agent/build/reports/tests/ s3://inspectit-oce-files/test-reports/$TRAVIS_JOB_NUMBER/agent/ --recursive; fi
+  - if [ "${TRAVIS_REPO_SLUG}" = "inspectIT/inspectit-oce" ]; then aws s3 cp $TRAVIS_BUILD_DIR/inspectit-oce-core/build/reports/tests/ s3://inspectit-oce-files/test-reports/$TRAVIS_JOB_NUMBER/core/ --recursive; fi
 
```

Comment:
Could you add an echo which prints the complete link? Or is it printed automatically?

Reply:
Done in the new version.